android/src/toga_android/widgets/selection.py

```diff
@@ -87,3 +164,9 @@ def rehint(self):
         self.native.measure(View.MeasureSpec.UNSPECIFIED, View.MeasureSpec.UNSPECIFIED)
         self.interface.intrinsic.width = at_least(self.native.getMeasuredWidth())
         self.interface.intrinsic.height = self.native.getMeasuredHeight()
+
+    def set_font(self, font):
+        self._font_impl = font._impl
+        tv = self.native.getSelectedView()
+        if tv:
```

Comment:
As above - how/when is this branch *not* triggered?

Reply:
During the creation of the widget, I saw that set_font() is called and tv was null